app/data_models/list_store.py

```diff
@@ -208,11 +210,11 @@ def add_list_item(self, list_name, primary_person=False):
 
         return list_item_id
 
-    def serialize(self):
+    def serialize(self) -> list:
         return [list_model.serialize() for list_model in self]
 
     @classmethod
-    def deserialize(cls, serialized: list):
+    def deserialize(cls, serialized: list) -> Any:
```

Comment:
This returns a store.
```suggestion
    def deserialize(cls, serialized: list[Mapping]) -> ListStore:
```